plugins/DevicesDetection/DevicesDetection.php

```diff
@@ -49,4 +50,33 @@ public function extendVisitorDetails(&$visitor, $details)
         $visitor['browserVersion']           = $instance->getBrowserVersion();
     }
 
+    public function renameUserSettingsModuleAndAction(&$module, &$action)
```

Comment:
Can we remove it again in Piwik 3.0 maybe? We could add a reminder to remove it in `DeprecatedMethodsTest`